script/hardhat/fork/govUpdateTest.ts

```diff
@@ -0,0 +1,326 @@
+  supporter = await initMainnetUser("0xc444949e0054a23c44fc45789738bdf64aed2391", ethers.utils.parseEther("1.0"));
+  governorAdmin = await initMainnetUser("0x1c2cac6ec528c20800b2fe734820d87b581eaa6b", ethers.utils.parseEther("1.0"));
+  aclAdmin = await initMainnetUser(NORMAL_VIP_TIMELOCK, ethers.utils.parseEther("1.0"));
+
+  comptrollerProxy = await ethers.getContractAt("Comptroller", COMPTROLLER_PROXY_MAINNET);
+  const Timelock = await ethers.getContractFactory("Timelock");
+  timeLockFastTrack = await Timelock.deploy(GOVERNOR_PROXY_MAINNET, TIMELOCK_DELAYS_MAINNET.FAST_TRACK);
+  timeLockCritical = await Timelock.deploy(GOVERNOR_PROXY_MAINNET, TIMELOCK_DELAYS_MAINNET.CRITICAL);
+
+  const aclAddressPadded = await ethers.provider.getStorageAt(COMPTROLLER_PROXY_MAINNET, 40);
+  const aclAddress = await ethers.utils.defaultAbiCoder.decode(["address"], aclAddressPadded).toString();
+  accessControl = await ethers.getContractAt("IAccessControlManager", aclAddress);
+
+  // Give Permission to Timelocks
+  let tx = await accessControl
+    .connect(aclAdmin)
+    .giveCallPermission(COMPTROLLER_PROXY_MAINNET, "_setLiquidationIncentive(uint256)", timeLockFastTrack.address);
+  await tx.wait();
+  tx = await accessControl
+    .connect(aclAdmin)
+    .giveCallPermission(COMPTROLLER_PROXY_MAINNET, "_setLiquidationIncentive(uint256)", timeLockCritical.address);
+  await tx.wait();
+  governorProxy = await ethers.getContractAt("GovernorBravoDelegator", GOVERNOR_PROXY_MAINNET);
+
+  //Deploy new governance v4 impl
+  const GovernorBravo = await ethers.getContractFactory("GovernorBravoDelegate");
+  const governorBravoV4Impl = await GovernorBravo.deploy();
+  await governorProxy.connect(governorAdmin)._setImplementation(governorBravoV4Impl.address);
+
+  // Iniitalize impl via Proxy
+  governorProxy = await ethers.getContractAt("GovernorBravoDelegate", GOVERNOR_PROXY_MAINNET);
+  tx = await governorProxy
+    .connect(governorAdmin)
+    .initialize(
+      XVS_VAULT_MAINNET,
+      PROPOSAL_TYPE_CONFIGS,
+      [NORMAL_VIP_TIMELOCK, timeLockFastTrack.address, timeLockCritical.address],
+      await governorAdmin.getAddress(),
+    );
+  await tx.wait();
+};
+
+describe("Governance v4", () => {
+  const NEW_LIQ_INCENTIVE = parseUnits("2", 18);
+  describe("NORMAL VIP", () => {
+    const proposal = {
+      signatures: ["_setLiquidationIncentive(uint256)"],
+      targets: [COMPTROLLER_PROXY_MAINNET],
+      values: ["0"],
+      params: [[NEW_LIQ_INCENTIVE]],
+    };
+    let proposalId;
+    before(async () => {
+      await loadFixture(governanceFixture);
+    });
+    describe("Propose:", () => {
+      it("should revert if below votingThreshold", async () => {
+        const { targets, signatures, values } = proposal;
+        await expect(
+          governorProxy.propose(targets, values, signatures, getCalldatas(proposal), "VIP 1", PROPOSAL_TYPES.NORMAL),
```

Comment:
nit: This would be a little clearer if we labeled who was calling. Here we assume we are connected to the default account and that doesn't have enough XVS staked. This isn't exactly true though because we are using impersonate account without stopping the impersonation